.github/workflows/gnoland.yml

```diff
@@ -57,7 +57,17 @@ jobs:
         working-directory: gno.land
         run: |
           export GOPATH=$HOME/go
+          GOTEST_FLAGS="-v -p 1 -timeout=30m -coverprofile=coverage.out -covermode=atomic"
           make ${{ matrix.args }}
+      - if: runner.os == 'Linux'
+        uses: codecov/codecov-action@v3
+        with:
+          token: ${{ secrets.CODECOV_TOKEN }}
```

Comment:
Do we need to use the Codecov token at all, considering this is a public repo?
I believe coverage should work fine without it